packages/dom/src/core/sensors/pointer/PointerSensor.ts

```diff
@@ -145,48 +194,38 @@ export class PointerSensor extends Sensor<
       }
     }
 
-    const ownerDocument = getDocument(event.target);
-
-    const unbindListeners = this.listeners.bind(ownerDocument, [
-      {
-        type: 'pointermove',
-        listener: (event: PointerEvent) =>
-          this.handlePointerMove(event, source, options),
-      },
-      {
-        type: 'pointerup',
-        listener: this.handlePointerUp,
-        options: {
-          capture: true,
-        },
-      },
-      {
-        // Cancel activation if there is a competing Drag and Drop interaction
-        type: 'dragstart',
-        listener: isNativeDraggable ? this.handleCancel : preventDefault,
-      },
-    ]);
-
     const cleanup = () => {
-      setTimeout(unbindListeners);
       this.#clearTimeout?.();
       this.initialCoordinates = undefined;
+      this.source = undefined;
     };
 
     this.cleanup.add(cleanup);
   }
 
   protected handlePointerMove(
     event: PointerEvent,
-    source: Draggable,
+    doc: Document,
     options: PointerSensorOptions
   ) {
+    if (!this.source) {
+      return;
+    }
+
+    const ownerDocument =
+      this.source.element && getDocument(this.source.element);
+
+    // Event may have duplicated between documents if user is bubbling events
+    if (doc !== ownerDocument) {
+      return;
+    }
```

Comment:
Note, new safe-guard to account for an edge-case where the user bubbles the `pointermove` event to the parent document (as I am doing with Puck)